src/spectral/rulesets/.defaultsForSpectral.yaml

```diff
@@ -244,3 +244,17 @@ rules:
       - $.servers[*][variables][*][default]
     then:
       function: truthy
+  # ensure that examples name does not contain space
+  examples-name-should-not-contain-space:
+    description: 'Examples name should not contain space'
+    message: "{{description}}"
+    severity: warn
+    resolved: false
+    formats: ["oas3"]
+    given:
+      - $.paths[*][*].responses[*][*][*].examples[*]~
+    then:
+      function: pattern
+      functionOptions:
+        notMatch: '[a-zA-Z0-9]\ +[a-zA-Z0-9]'
```

Comment:
I don't think this would catch a space at the beginning or end, like " example_name". Would just `'\ '` be sufficient?

Reply:
Maybe this: `^(.*\s+.*)+$`

This regex is meant to positively match against any string with a space, and `notMatch` should negate this such that it fails when the string includes a space.

Source: https://stackoverflow.com/questions/9341379/check-string-contains-whitespace-along-with-some-other-char-sequence-using-regex